unittest/resources/test_auth.py

```diff
@@ -41,3 +41,62 @@ def test_login_get_success(self):
                 'email': 'test@test.com'
             }
         }
+
+    def test_logoutGetSuccess(self):
+        """Verify user is successfully logged out.
+        """
+        user = User('Test User', 'test@test.com', 'password')
+        db.session.add(user)
+        db.session.commit()
```

Comment:
It seems like in almost the tests we are creating a user right? We can put that in our `mySetup` method once and it'll execute before every of our tests.